src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -1364,13 +1364,6 @@ public Boolean visitLiteral(LiteralTree node, Content content) {
                     return false;
                 }
 
-                @Override
-                public Boolean visitSee(SeeTree node, Content content) {
-                    TagletWriterImpl t = getTagletWriterInstance(context);
-                    content.add(t.seeTagToContent(element, node));
-                    return false;
-                }
-
```

Comment:
Did you remove this method because the enclosing method, `commentTagsToContent`, is called only for a series of inline tags, which `@see` is not?

Reply:
Yes.  See this text in item 4 in the description:

>At this time, an "impossible" case was removed from HtmlDocletWriter.commentTagsToContent. The method is for handling text and inline tags, so the visitSee method to handle an isolated @see tag can never be called in practice.

Reply:
It was only after I looked at the changes and asked the question that I finally read the description; sorry about that.